src/main/scala/net/degoes/02_ops.scala

```diff
@@ -472,7 +472,8 @@ object education {
 
   final case class Checker[-A](points: Int, isCorrect: A => Either[String, Unit])
   object Checker {
-    def isTrue(points: Int): Checker[Boolean] = Checker(points, if (_) Right(()) else Left("The correct answer is true"))
+    def isTrue(points: Int): Checker[Boolean] =
+      Checker(points, if (_) Right(()) else Left("The correct answer is true"))
```

Comment:
fyi, there does not seem to be a meaningful difference right here. Maybe another push?